components/molecule/autosuggest/src/index.js

```diff
@@ -0,0 +1,245 @@
+/*
+ TODO
+ - comprobar que el foco de la lista va bien
+ - se cierra el listado sugerido si se hace click en cualquier sitio de la pagina
+ - la lista de sugerencias, si un termino es mas largo de lo que se neceista se truncara
+ - sort funciton personalizado
+ - listado con sugeridos recientes. Guardados en la sesion, se pueden borrar y limitados a 3
+ - listado con terminos personalizados. Se anaden a la lista de sugeridos tambien
+ - multiselect con etiquetas
+ - que se puedan configurar los iconos
+ - cosas del responsive
+ - sass
+ - add new model as input
+ - cambiar a funciones stateles
+ */
+
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import OptionShown from './model'
+import MoleculeSelectList from './list'
+
+const BASE_CLASS = 'sui-MoleculeAutosuggest'
+
+const SIZES = {
+  SHORT: 'short',
+  LARGE: 'large'
+}
+
+const LIST_BEHAVIOURS = {
+  ON_FOCUS: 'focus',
+  ON_2_KEYS: '2_keys',
+  ON_3_KEYS: '3_keys',
+}
+
+// const TYPES_
+
+function isoString (str) {
+  return str.normalize('NFD').replace(/[\u0300-\u036f]/g, '').toLowerCase()
```

Comment:
We need to support IE too
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize